packages/cron-job/Tests/MessageHandler/ExecuteCronJobMessageHandlerTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Component\CronJob\Tests\MessageHandler;
+
+use Draw\Bundle\TesterBundle\PHPUnit\Extension\SetUpAutowire\AutowiredCompletionAwareInterface;
+use Draw\Bundle\TesterBundle\PHPUnit\Extension\SetUpAutowire\AutowireMock;
+use Draw\Bundle\TesterBundle\WebTestCase;
+use Draw\Component\CronJob\Entity\CronJobExecution;
+use Draw\Component\CronJob\Event\PostCronJobExecutionEvent;
+use Draw\Component\CronJob\Event\PreCronJobExecutionEvent;
+use Draw\Component\CronJob\Message\ExecuteCronJobMessage;
+use Draw\Component\CronJob\MessageHandler\ExecuteCronJobMessageHandler;
+use Draw\Component\Tester\MockTrait;
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+class ExecuteCronJobMessageHandlerTest extends WebTestCase implements AutowiredCompletionAwareInterface
```

Comment:
This package is stand alone (not a bundle) so you cannot rely and WebTestCase and AutowiredCompletionAwareInterface

You will need to add your dependencies (require and require-dev) in the composer.json file since the CI will execute the test of the package independently.

You can rely on draw/tester package in your require-dev which provide the MockTrait